### DISCUSSION TITLE
Doc: Outdated doc

### DISCUSSION THREAD

**Discussion by Romainlg29** [pmndrs/zustand#2627] [Bug report] [answered]
### Bug Description

Hello,

I am maintaining a middleware that sync tabs and browser windows using the Broadcast Channel, however someone asked me to make some of the internal properties accessible to the user.
But, looking at the [documentation](https://docs.pmnd.rs/zustand/guides/typescript#middleware-that-changes-the-store-type), the recommended way to create a middleware is to use a type that is marked as deprecated.

I still looked at what the other does, but I'm getting an issue I can't resolve about the types.
```ts
Argument of type 'StateCreator<CountStore, [], [["shared", StoreApi<SharedState>]]>' is not assignable to parameter of type 'StateCreator<CountStore, [], [never, unknown][]>'.
```

Is it possible to update the docs to match the current Zustand version?

### Reproduction Link

https://github.com/Romainlg29/use-broadcast/pull/9

**Comment by dai-shi** [comment]
If I understand it correctly, what's deprecating is `State` type.
Can you just remove `extends State`?
If that works for you, can you open a PR to fix the doc?

**Reply by Romainlg29** [reply]
Yes, State is marked as deprecated, however removing it doesn't help with the type issue...

```ts
Argument of type 'StateCreator<CountStore, [], [["shared", SharedState]]>' is not assignable to parameter of type 'StateCreator<CountStore, [], [never, unknown][]>'.
```

I've the following types...
```ts
export type Shared = <
	T,
	Mps extends [StoreMutatorIdentifier, unknown][] = [],
	Mcs extends [StoreMutatorIdentifier, unknown][] = []
>(
	f: StateCreator<T, [...Mps, ['shared', SharedState]], Mcs>,
	options?: SharedOptions
) => StateCreator<T, Mps, [['shared', SharedState], ...Mcs]>;

declare module 'zustand' {
	interface StoreMutators<S, A> {
		shared: Write<Cast<S, object>, { shared: A }>;
	}
}

type SharedImpl = <T>(f: StateCreator<T, [], []>, options?: SharedOptions) => StateCreator<T, [], []>;
```

**Reply by dbritto-dev** [reply]
@Romainlg29 what about this -> https://www.typescriptlang.org/play/?#code/JYWwDg9gTgLgBAbzgWQK4wIYwKYBo4DKmOAwlNltPkdNgIJjDUy1rHQCSAJtgHYzAAZsGxQ4AXziCoEEHADkAL1QBnTLy7yA3ACgdMAJ5hshFuQIALDOS4AeAgD44AXkQ64HuAHoAVD-eecD5wHIJwMBbAKuEYAEbecADuwBoQiXBR4RYmIBgpMfFeSSlcaQGePl7lGSrIebwAXHCxEBAANhS8OuJ62AAekLDhRiaW1thcAPJgAhC80a4I1b7+gUFwACrZcLwYICYQYREmsTIYXADGGGpwF1a8vNht1esc8CCq8LEmqLzAAI6obAvSrVXb7AD8TTUUBSAHNdMs-CCUPVwqBsBB0CjQlkcmjMIViql0qVsCpePJ4OQVJANBleFlMgJ9vgIsy4glkiTim02s1sBdZCZjnBcikUQABHiCDCoNrwACMAAZlWKVCCqoFxbwNhisTAoTtUCBvlBEYEVjijlAgWztmpaLz+XM2gY4CoDLw7jI-ooDt6TMkInixQS4gA6OB0QQ4MQRLD2kyO8jOgVwX6e70WX3Af1cCNSmVyhVSDBtFTAtagwKZr0XI0tdqdXTiRGGYxwADqsJwtg2+AAqk5XJMQMAYP38ABrbAGQ5wYdwABki-bI27E4sYxs9hHhDg-RwGmiSDh2BgRCw2CaAAoAJQuJwpQSiTYSaoQ7u97D2Zi0HcJn7BwHGqJpHgAN1ERErXWLZRisGxhmMdxQX6QZ4A7BDxi4Fw4FsaoB2qZAwGiI8+C4aIAG0aHINhKCgbg+AEYRRHwX5p14NJeAAXSoni8P43BiIuMi+mPSi4BosxsHoswmP4IQRCgdjeE47i+IE1x+J0Bxb2qQQmivUhyAYqcpIjSySJUfAqPkFREImeR8Eg0QeJ4-BkFEhxhMCCAZmAOYVCNQCpgCoKdEfZwnGM7AyAoMxzOs2y7IcnDnJ2bAoKgDy4EsiMvJUHiHBg5F1gAUXEijQwALU+DB6RTKtQR4C42nGMUIC4eUTCUeqNC8CCGuAPkMHkNxAhSONZQuUYZLk6AVD-aMnCWNZPDSmwmi7LdQr3aoeh6HRYOCDYN1AMAOn2fgsECxkF1FUKpF+C5Zi6IIqiwwhHK4DhwH5VxgNvQzTGveKzIHKTcv4ny4H8t7gqMn7pgRqKYuIOLTMSyGhKhkq9BO76cIyf7sGu4g7tQuBJTAaw9ikOA6rUBrcOZnBbix6AqZpum5Hhu7onguHwvmVCqiFeZ4E2iY-supGcNlgG4GB-B+aCtHlcrGB8HPbW4AAfSajW1s8IRlawhduVKdJnFthRfhlFInMfE31ollRmwjRJrF4W95AAITOS5rhgEh7keflMi4qXUDADCJgZJkyN4CDgF9cmoyFpq02j9NpYLeR710daPHIGBUCgRlBFvLWdYvfBDZkouDr0S0ysCYJcUDiBziuNQw4aiOah2CAY7j6AcC4fAy4rxlRVia5gAuD0ZM1aozdvC2wm73uQ4Hh4nhcO35Ad7BhEeTQXZeTx3c973K79nfg-78PD6j0ePVj+PcPydlolONIlYoCZwdDJHOH9vgeh+hGQuxcS4z0rlIGu9c4C6wbkbOBHgjpt1WB3TYIwNTVi1J4L6BA8IACULyzzOsYWwW9UEXnxoEUhHA8IAAVeYXlEEtWK4NEoEBSgAInzoI6g7lbLFQcFRAATDxTByETBvDJnhJAVFZwGGhDAWEvA4Q8SaBxLiiRGRtmqF9ZA5IVAYHPC4a+HgAA+E0S6BAwK9O6TR7L1m0LYj8TiHGuycXAFxb13F3AaueLxASNoYyaEokA8jAg9F8Y4yJQS3EKHOFwfWjxEj60JBEyJGQuDgRNGaeJnhEklz8d45xri5ghLaBASs+TInACKcaU00FvEVPWlUgpgTamNAUKEnR2B9Y6maQE1pxSOnmmqSQuIKhplmn4mUrBFob5BSlmA1wjcnTXEIBwdZHhYJ4I4NEUU2csyzVwsGCwoZR7ZHjAstegQOjwCiAQesCdXC6wfHAAAhHbU+58JhHO8O3CoIRzmRHOZyIoVs0ihh1AULkJQyjVm7NkRkGBMrpEJKinkmR-J8AmPgCcaZIFXITrcpF9QXmeDeTUOo+RXCygrNgMFJzIVCwAbvG4wyh7girB3YhHh3bwCFXhNW8wIQRklRCL8ggIwsCINouED5OUQo8KdbYrS4ZHG2Pi+FaLEj0o8IyvVrhlSatwZC2iJgcVtCiPABc5Z+SikJNEY1JJCF4MmLwN0UhoC0t-nC4k1szW3E2QURZ7TllaSksqORrdPBcu1XAPhIpti8ufvAAVTxI3ituK-JW2S4BPz7qHEtt4hXNyRLa9NAAJFmHRar9VZheZ63o3oog2LCOE54xA4sgGocxKhLHWJYIEvk9zjhPNiL6iooqo2SzhrwAgF4mj0K1nhW8+VrBwhUMbbxaaMUAHFO2ijAOQVOWJohs3JNUmsJci3XqyoFVQeFfmPn2bEsFOCn2LjAFwa8oYH2LvWs+9aWs92WQPUe-9qatUYtxBcmS96LBYjaLhXOlKvlTzgAg3ghYAlQbWBvaVwUIx1m9FfPpRHVkfhPchvBF7MLbGkOSO54Gn3LsCEW2OIHJ5fovH839OA4kpvWqe1jl7QHXmhVgYtYSJi8e8UWh9eFJixAAFaChgBGZisJyS3kE9eLg94Iw2FQLNW8t4IC6dsuo-AQ02g8WPX0jeb7b2qDUXOASgLXCubo30zwDmdMqLypZcLTn-NNFcxIRjCS5mlyoYg8LjHxD4AQJIcTZM61OJk3a6qaGFPhAgNO912wHlvk9Wppx+a2gRhHTAMdE7sC3iQKkupQz7jhPwJp3L0Q2tWOwM3BJNqUQEFQAui4sJIFTvnkHStymD7PCIdURrEY5j7HHaN3dY2nzJPI2EOz2AIxCZxfskb55LPdcZLbVwHjvSF2O4VljJdO4Go5ESBF6R37sfxCkaeaX3p9LI04je-yojMt4CF0LhHQdJfKVJgJRXINwEoSACAUFKuhkEC9BGgTGoGFNO0aI0hZBgdXqFiHL7o2adHLp-Thn+DGZULeb9gToixMs9Z2z9nHNSWc3AVz7mjv+M86d+hCXAsKAJ92u641lyrhl+WAFx9PRk7aK9yXCO4a6ci-lGLwu4ui-VyY-XPj9dEYNzp5HWDsviAKwU9HGKN30lK+zRb1W50xpS9qvjL6S3Ncaa1ixo3Ov9OCb1lTGVBvc5QBH27iHpMfYxxuwHuKUXeutoUgPH0A9FuyfrS1MaqKeojB0HRIYAC0cBFQ8X+XAAA1A3h3MbmuqAsDW7AOTWku8iVtlrN2OtdYGe4jJWS++5LiBlKZuLS+4SG0nvbKfUdOIY10jfSGG3u87Q1QpQb4yGrDX9jIYRyVkgpFSOAVhcdzGFZ9oPHgN63nO5dxPo+7sDKPk9qfJeeSK4q40OtQaIKuR+j2cAyo8OkyuErgH+WAGAEYrSHeW+Ti2CJcbuwQ-qiOs0wAqcOigS2eD67EwGoGXuT+kGL+F+ys7+F2SBX+yeY2EY92f+seIyuuAebueCg45B3u8mOABedO0GomiBmAEYD6g+aO6e++WeVKuEWAWiwAsQ6AVBpGNBk0KgnygY8B4QtoHK2+zGe+fqjI5A+BhBcIxBbUjSOQzBIO2OuOHqZ+JqR+lOcgooTqag9WJcb+4hV2w2zBP+b07B8gNhTSsBkOp2oBsOkRBS6BBSmBQ+0aJQ-QeEleqRfQkw1c-hKBFmHeG8mRGurgteiocRkSletITqtmmR+AZRHe3SawSRaw2BcAuB5h2ABB8I1hfWoyyKu27W+Awyk62wyKRq4a6KTiIhtBZ2DBmATBa+LBbBUBYRvRYy9QXBUuysuRlqdsA+b2BSMOaIrgWiQIHezCCykhl0S8HWyobIlx1ecIEQ+A+UuRnq42iR2+E2qOp6wQOhPocweYQYW4Iac8cQhaDO9YOYgJ-ou6948WEAlqq06mIeI+zBUe927iVy40K+3+qe4KJhkKuIHy+Gw8bKlYgSsYb4KoyoIANkScKKzIox4YsQPhohMAeo+wBodmHmhx0RJJuh5RkyYBLK+hZxBeps2hpJJxBhDRXSqsIsVGOonJmI6AcACqDeqoHxayPxyGwQzaGgrafcQYWKoY4x5+mQ4RqmG2-G0acwJADSFJrgD4CJSJBxYqqJYeo+GJE+Qyjp2A8+y+Cx7W0hAGpGIQYQiQAZqY4R3RooOo+AuQs4YGgocwuEooZaOobJJ2ysRxKQQpYZ4OEZeIqYRKAa7oj+KKHQsYQx-poYjWxOuERGwhmhps0uDxfATxdyUBZR7pMhhJkSwQDpnQGYYAEpS645q2EcEYVpGqk5CRkSzRDW0a5AOo8Is+C66RlxwgCoogdmhIGs+KcuA+Hew+Xp6J4+MeqxKm6xKQgZTQq59Q65nqVESa9xC6D5ZMT5OiG50QuJQR+JgQC5TRB0k2GKdAXAuEVph4UE-AcAXhx4ogkaf2rBkF5UsFMAAAMs6iSlAH7N8IILQL8A0ucBlPaf6dIb8YQFCbmLCTSvGSyZGlDgKdckKVctCX6B1tqUxoWZCsgF1EIO6CzIjuXIgqKEzOoKzDTiKtUE1JIReLFFKuuheGCrbtXHMJnnXHrBgt0KVKsH8T9GKK0lwB0PfM1NzP0HsJdM1CTBhIgBzAlCYJIO4QoMoMzBoBEsEBdBPPZfnBIFIDIHICfJWLXjmpWrXjACoJ5VTAJpWPanhHNo5fpOsGsPnMlcEAEsgjABrFHkKL8DAE0GqM7r5BlU4kgEKu4iAAYLXk1DiVTGsPeFTONmhAMD5RpoZa4PnIrIngYtxInqFFoEAA

**Reply by Romainlg29** [reply]
I'm getting a never instead of the SharedState from above.

```ts
Argument of type 'StateCreator<CountStore, [], [["shared", never]]>' is not assignable to parameter of type 'StateCreator<CountStore, [], [never, unknown][]>'.
```

**Reply by dbritto-dev** [reply]
@Romainlg29 I don't get any errors -> https://www.typescriptlang.org/play/?#code/JYWwDg9gTgLgBAbznAyjAhjApgYSlzaAGjgGN9MsS1osBZAVwxmgEkATLAOxmADNgWKHAC+cPlAgg4AcgBeDAM4Yu7GQG4AUJpgBPMFlQt8KABbp87ADwoAfHAC8iTcmQB6AFQeXruB7isfHAwpsCKwegARnBucADuwKoQcXBhwaaGIOiJEdGxCUlxPq4ebsWpinTZXABccJEQEAA2BFyaItpYAB6QsMH6hmYWWOwA8mC8EFzhTgjlnt6+fnAAKhlwXOgghhBBIYaRkujspOjKZOZcXFhN5cus8CBK8JGGDFzAAI4MWHel5ZttgB+OrKKCJADmWnmXj+cCqOV42wgTDhgXSmWquRi8US7GScHxWEUXBk8HwikgqlSXHSaSRVDp4QweVxhVxTSa9SwpCkhn2cCyiThAAFOHx0AwmvAAIwABjlgsUfzKviFXBWoCwKJgII2DBAryg0N8CzReygPxIAuUtA5XKmTV0cEUui4pFMkg+ch27sMCRCGMFWJZADo4ABBPjYYQhTDW9a2-D27lwd6u92eqbAH3sUOi8WS6XidBNRS-Jb-Xzpt2kPUNZqtLQiaF6AxwADq4OwVhWJAAqvYnKMQMAYL2SABrLC6XZwQdwABk89bA07Y9MQ0sNiHqDg3WwqnCSAhWBgaEodQAFABKRz2RJ8ISrUTlIGd7tYGzUYxYLcjXtbFsco6muAA3IRoTNZY1kGcxLH6AwXH+bpengNs4OGdhHDgKxyj7co6DAcID24dhwgAbRofBGGYNhOB4fhBCgEh3knLhki4ABdCiuJw3iiEI0gSK6Q9yLgKjf1owgoA4bheAEIRWK4djOJ4vinF4zRbCvco+DqC9sDwAhjAnCTQwsojFBICiZEUeCRhkEhwKELiuJIOhhNsQTfAgCZgCmRQ9X-MZ-MCzQ7wcexDNwChTL7eFiJs2z7KwpyNiwCCoHcuALNDTzFC42woNhZYAFFRLIoMAC1nnQakkwrf5OFIJphkFCB2ClQx5Dq1Q3DA+rgE5dAZGcXxEhjCVSEGKSmBkxRv0jew5iWVxUssOoOw3EKd3KDoOk0aD-BWNdQDAFpth4TAAtpOcBRC8R3lISY2j8MoMNQBz2FYcAuScQCr30oxKGMmSzIEiSipIPzXqCgzvvGOHIui5hYpM6AIZy3jiu0Y6vqw1I-qwK7mFu5C4BFMALC2cQ4Fq5R6uwxnsDIOLoApqmaekWHbvCWC4F58L3s0XlpngDaRl+i6Eaw6X-rgIGYbC6YUcV8sYBIU9NbgAB9Rq1dW1x+EVjC5wKfEUgca3ZHecVEkcu8jbWsXFEbUM4gsLgrxkAAhI4TjOGAcEua4uTSDiJYYMA0JGGkmX3LgwOAL1SfDAXGpTSPU0lvMZBvLQ1uQfAYAYKBaT4K8Na1s8SH138C-27RTVK3x-HRf2IGOU5lBD+qw4qDYICjmPoGwdgSBLsvaQFSIzmAUgXV-FVyhNq8zaCTvu6DvurhuRwbZkO2sAEa41Cdu5XFd93PfLn2t8D3vQ-3iPh5daPY+wxFQnCQ5knLKA6dEy-izm-V4LpvqhnzoXIuU9y7iCrrXOA2s64GxgcgQ6LdFht1WAMZUlZVSuE+igHCAAlM809ToGCsBvZBZ5ca+GIawHCAAFbmZ4hCLRimDUyKBkoACJc78OoG5GyRVbAUQAExcXQYhQwDwSY4SQBRacuhQQwHBFwCEXE6hsQ4nEWkLZyifToMSRQ6BTyOEvsgAAPuNIuvh0AvVunUOytYNDWNfA4uxzsHFwCca9VxHp6qng8X49aaM6gKJALI3wHRvH2PCQElxshjjsF1tcOIusWRhPCakdgoEDRGlia4eJRcfGeMcc4qYQSmgQHLLk8JwACn6kNJBTxZS1oVLyf46ptRZDBK0VgXW6pGl+OaYUtpxpKlEKiIoSZRpeIlIwSaK+gUJYgKcPXO0ZxUCsFWe4VuJQAjMmAXaDMM1sIBlMEGYeGRYxzJXr4Fo8AwgoFrHHJw2tbxwAAIQ22PqfEYByYhHOQO3U59Iog4gtgSAU6psT5DxMkOEHYMi0nQBlFILIYXIpSGkPy3ARgkDHCmcBFy47XKDOqJ5rgXkVARLSJwEoyxYBBdBHBAs-7b3OIMgegIKxt0IcgV28ABU4SFtMIEoZxVAnfHwUMLA0CaIhLedlYKYLrGaYLPY6wcVIsKLS5A9LtVODlOq7BxzqKGExU0MI8A5yli5AKFk4QDWW3wTg0YXAnTiGgNSkM0L3UooIeUUVuR5mtMWRpCScoZHN1cBy45PD+TrG5Y-eAfKbhGrIOsi4-d95OEyXAB+Pdg7PyaFeAVjcYSWvBXAAAEkzFoNU+rMzPE9d0r04QrHBBCU8whMWQGUKYxQ5jLEsH8ZyW5+wHmRE9SUYVubxaCy4CgM8dRaEaxwlePKFgISKENp4pNa1-AAHEO0CjAPgZOKJwgs2JJUqsRdw3XsygFJQOFvl3l2dEkFWCn3zjAOwSgQYH0LtPUu3wGtd0WX3Ye-9iaNU4PRDaX897TAoiaNhbO5KPkTzgHArg+Y-HPrWmvSVQVQw1ndBfHpRHlmvmPch45F70LrAkMSG54Gn1QbWSu6OIHx5frPD8392AYkJrWieyscA2NgbRqczA+ahnsF4548ND6cKjEiAAKx5DAUM8lwTEivIJyg7AbyhksAwGaV4rwQD0zZVRJBBpNC4kenpa8323qUComcfF-lODc3RnprhHO6aUblCyEXnMBbqG50QjG4kzOLhQ+BEXGMiBIAgMQ4mSY1ocTJnB67qRocoMyCAU7nXrDuc+V16mHFZqaKGYdMBR3jqwFeJAySakDMuKEkgWm8vhA6xYrAjc4kWrhCgBg87yDAHAZO2eAcy0qbDjm5roYpjbDHeNndE37yJKWGvK8WBQxCcxbssbp4rO9aZYfC5+djtFZY6egIuqoWslhfi8I2d4XVEnult6PSyPjKCFeX5YRGWhbC4R4HyXSlSb8cVou-hyEgAgBBarQY+DPThv4hquhDTNHCBIKQCnaCpfBXxl2eatPDj0wZozPATOKCvN+-x4RolWZs3ZhzTmJIubgG5jzR3fFeYh7QxLQXZB467bdMai5lzS9LH8x7xOGhNGexLuHgs9NRbyrFoX8WRdq6MXrrxeuiP6904jjBOWRCFbyaj97pXsLldZst2rs6I3U5Fj0rbbWbtdZ630oJA2sDpWG1z+EZjxvO-Ca72T672OGGLfqtklt8n+7B01vNmTdamojRRV1oYWhaMDAAWjgDKLivy4AAGpa-24ja1pQpgq1YCyc0xPfig-1Pa-H083XemBNSewdJheckkpaYX7VI2497du4htaDGOnI4A6R1AHb6r5L9bGPVQas8EhNqSokJIyRwHMNjqYgq0e09SBDs7F3MBXdG8PiboZ7sHycDINJGS3e2SUQSuy4UOlQWIyu++1sZqsO4y2ETg52l2oYzSre6+DimCRcye-g3q8OM0wAycWi-iWKS8lArEwGoGnu9+kGnip2SBb+seIed2fSv+-WISUecBr2da2+-YFBXuwClAuej+0Gom9BGAoYD6feXBueO+aeLo+G-iMAGii2TA1BpGwhxsig7yfoCBwQlobKG+zG3BXqtI+A+BhBEIxBrU9SmQn+QOmO2OLqx+P2++5O0gAodqygjWRcdBr+GAjBn+zBr0rBMg1hDSnBPhEO4BMOL2eS6BeSmB4S4aeI3QOEZeKRXQowlcYh6AKBlmrea8GR6uTgVeMoER4SZelIdqdmGRJAZRrenSSwiRSw2BcAuBZhWABBkIVhkeIyWIu2nWJAgyE66wCKmeP23h5Gz+ORARy+X+P+MBbBQyfRiQOu1Oa8OReRrBvesRTSEBOQTgGiPwrejCcyEhF0C8XWco1oZxFeEIIQJAeUmxrqk2CRG+U2yOMm-g2hHoXoOY-oG4AaiIUQm29OtYWY3ohgTgt4CWEApqK0GmFarWg+Ieo+92riT2ogsxnWUhSGxhxy6IbyChaQLK5Y-i0Yz48ocoIA1kCcOK9IoxgakQkxSwGsmoyITA9mnmeSa84BPxIw5R4y+xTKehxx-umh-JuhRxBhPSjRcSyscM0q6o7J2oTAcAcqteCorxKynxGq-gTaqgLaPc-o6KQY4xeKg8YRIwoJK6UwOAdSZJ0JN4sJ8Jux62NwyJI6n+aJ4eAyDpUes+WJ12gRq+oK+J9a6IcQUeyYYR3RAOiQJAWQ04YGPIUwHu6wxaNK6htBEO0O1QgpW+oOH2GIyYBKPqzod+2ILQ0YQx-pQYzWhO2ERGQh6xUutx3A9xNyixZRbp0hluyw9prQaYYA4pi6o57pLWVpaq458R4SzR+eK6+A6okIwB86aRZxAg0oQg9mLIasOKsuvereA+XpcxPp4+oRvRoys+dQS51QK5rqFEcaNx86N5JMd5Wiq54Qi+TBoZvgs5TR+002smEYk+ZAdZmU8kcAnhh4QgOaP23+k+ZUEEPAAAMvakSlAD7K8HwLQO8HUscOlHaf6X3l8agOCX8T6LiIGPGTPCCaGhNFEUSToYKRchCf8dOSlrWnCHQJ1PwM6EzPDqXPAgKAzCoMzMvPRRErQBIWeDFBKmumeCCjbpXFMKnjXDrGgu0CVIsN8d9IKM0uwC0LfE1JzN0FsBdE1ETGhIgGzCZIYGIG4bIAoIzKoGEv4OdGPDZbnFiY5UfOWFXummWlXjAIoG5RTOGkoH+JsrZZQLpMsKyd9HFf4H4ogjAGrKPryO8DAHUIqE7j5MlQ4kgAKq4iALoFXo1GNPEgVa4DeBTJNihD0J5ZpnpU4LnPLLHnopxLHiFKuO2DgCiDwNakopoAAJCZU8ALLtIjWJDkAkzyTXhqxgRwnsBaAjUtRLnzWKyLXLWrWY6cCuI-FjR2IyAAByb8h1q1GsPF+1ise1WAB17icAx1Z18Ah121qBWlnQjVfQEV5Y-VWVQ1Tgs1lAVg-1g1v4OkN4ukI1uc0NI1qV6l6VcwI1KN412VcA1xo1KNM1G1E1W1R2MGVcSNuaWVoIoYaNTeteogN4N4gkKNa1PIuN6NPyUULoomSAaNdQ365NA18ANeMo1NdNKNd1D17oTkWNMNZ41191t1nUh2rNMGSAd1gtWNeVWNRVWwMtMgpV5Vv4lVo1tVBcQAA

**Reply by Romainlg29** [reply, marked as answer]
Thank you!
It seems that the issue comes from somewhere else...
When bundling with `npm run build` and then importing it, I've the type issue.
